EIPS/eip-5792.md

```diff
@@ -40,13 +40,13 @@ The capabilities field is how an app can communicate with a wallet about capabil
 
 The wallet:
 
-* MUST send these calls in the order specified in the request
-* MUST send the calls on the same chain identified by the request's `chainId` property
+* MUST send the calls in the order specified in the request
+* MUST send the calls on the same chain identified by the call's `chainId` property
+* MUST NOT await for any calls to be finalized to complete the batch
```

Comment:
can you explain why you think we should enforce this? I'd expect this to be up to the wallet, until we have the other capability we talked about which would enforce waiting for calls to be mined across chains.

Reply:
My understanding is that we would enforce the behavior to always be "parallel" by default 

So the new capability would be "sequential" if available

But my take is that either way it should be deterministic by the spec rather than wallet-dependent

Reply:
Hmm just seeing this. I do feel like generally order matters to devs, but trying to sequence multi chain calls is probably too difficult. Too hard to know when the output of one will land on the other chain, if I am, say, funding my account on chain A from chain B